go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/ipfs/go-datastore v0.1.1
 	github.com/ipfs/go-log v0.0.1
 	github.com/keep-network/go-libp2p-bootstrap v0.0.0-20200423153828-ed815bc50aec
-	github.com/keep-network/keep-common v0.3.3-0.20200512154128-4cefc7b2f12e
+	github.com/keep-network/keep-common v1.0.1-0.20200515214626-9036c7c35946
```

Comment:
Do we want to set it to the commit where we merged the PR to the master? Current one is also fine with me.

Reply:
I'd leave it like that for now, we'll have to bump up `keep-common` version for the next release anyway.